examples/src/main/res/drawable/user_puck_icon_uncertain_location.xml

```diff
@@ -14,6 +14,6 @@
         android:pathData="M14,42.5C14,26.8,26.8,14,42.5,14S71,26.8,71,42.5S58.2,71,42.5,71S14,58.2,14,42.5"/>
     <path
         android:fillAlpha="0.7"
-        android:fillColor="@color/rawLocationUncertainGray"
+        android:fillColor="@color/raw_location_uncertain_gray"
```

Comment:
`user_puck_icon_uncertain_location.xml` is not being used anymore from the `examples` test app. Could we remove? One thing that would be cool though is to change it (e.g. the color so it doesn't match the default one) and showcase how it can be customized. What do you think?